src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -70,7 +70,6 @@ class Conf(args: Seq[String]) extends ScallopConf(args) {
     name = "staged-digital-object-set",
     descr = "The resulting Staged Digital Object directory (will be created if it does not exist)",
     required = true)(mayNotExist)
-  codependent(urn,doi)
```

Comment:
DOI and URN are both identifiers for the dataset. Actually, they are currently both required, so maybe we should add a validation that says so. Saying that they depend on each other is incorrect. That would mean that they could be omitted, if they were both omitted at the same time.